### PR TITLE
Whisper fine-tuning recipes for CHiME-4 and WSJ

### PR DESCRIPTION
**Pull request by YoshikiMas**
This PR adds configurations of Whisper fine-tuning for CHiME-4 and WSJ. I'll upload the pre-trained model later.

### CODE REVIEW

File: egs2/chime4/asr1/conf/tuning/train_asr_whisper_full_warmup3000.yaml
Comment (sw005320) [inline egs2/chime4/asr1/conf/tuning/train_asr_whisper_full_warmup3000.yaml:1]: If this one is not used, you can remove it from this PR.

File: egs2/wsj/asr1/README.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
 # RESULTS
 
+## Whisper [medium_finetuning](conf/tuning/train_asr_whisper_full_warmup3000.yaml)
+
+## Environments
```
Comment (sw005320) [inline egs2/wsj/asr1/README.md:5]: Ditto
Please clarify that this does not use an LM.

**Review by sw005320 — APPROVED** [review]
LGTM.
Please clarify whether you use an LM or not in `REAMD.md`

**Comment by YoshikiMas** [comment]
To clarify the point that the performance is with `--cleaner whisper_en`, I added a comment in each README.md. The use of  the cleaner is consistent with previous whisper fine-tuning recipe.

**Comment by sw005320** [comment]
Thanks, @YoshikiMas!

**Pull request merged**